### PR TITLE
Adds Durk plush

### PR DESCRIPTION
**Pull request by BeeRobynn**
## About the PR
Adds Durk plush to the game! Hi durk!

## Why / Balance
I'm being held hostage please if anyone ca-

## Technical details
Adds Durk to list of plushies, only as an admin spawn

## Media

https://github.com/user-attachments/assets/fdc5c164-5f6c-4a16-9799-fc35dc1154ed

**Changelog**

- add: Added Durk!

## Summary by CodeRabbit

- **New Features**
	- Introduced a new plushie entity, **PlushieDurk**, resembling an atmospheric technician, complete with unique sound effects and a playful description.
	- Added a state object for **plushie_durk**, ensuring consistency with existing plushie entities.

These enhancements enrich the user experience by providing more interactive and engaging toy options.

### CODE REVIEW

File: Resources/Prototypes/_Goobstation/Entities/Objects/Fun/toys.yml [hunk 1]
```diff
@@ -52,3 +52,33 @@
...
+      path: /Audio/Effects/adminhelp.ogg
+  - type: EmitSoundOnActivate
+    sound:
+      path: /Audio/Effects/adminhelp.ogg
+  - type: EmitSoundOnTrigger
+    sound:
+      path: /Audio/Effects/adminhelp.ogg
+  - type: EmitSoundOnCollide
+    sound:
+      path: /Audio/Effects/adminhelp.ogg
```
[the remark this answers is not in the document]
Comment (BeeRobynn) [reply Resources/Prototypes/_Goobstation/Entities/Objects/Fun/toys.yml:79]: hammers-this-guy

**Comment by Icepicked** [comment]
chat didnt we literally remove plyush's plushie over self-inserts

**Comment by Aidenkrz** [comment]
> chat didnt we literally remove plyush's plushie over self-inserts

chat didnt we literally say when we started the server we were going to accept all item contributions from layers like those, but a certain ex staff member decided they hated one particular user and forced the maintainer to remove them?

**Comment by Icepicked** [comment]
every action that G did is somehow based out of pure hate for a certain person. 

No, it was a highly agreed upon action among the admins to remove plyush's plushie and to deny their at the time new PR that was adding more self-insert stuff. Not every single ban by G was out of hate for someone or something.

**Comment by Aidenkrz** [comment]
> every action that G did is somehow based out of pure hate for a certain person.
> 
> No, it was a highly agreed upon action among the admins to remove plyush's plushie and to deny their at the time new PR that was adding more self-insert stuff. Not every single ban or action by G was out of hate for someone or something.

It was a discussion between you G and Natalie, and G was yelling at Piras to remove it before you even brought it up.

**Comment by CatMagic1** [comment]
haha drama

**Review by whateverusername0 — APPROVED** [review]
/Audio/Effects/adminhelp.ogg

**Comment by whateverusername0** [comment]
fix checks

**Comment by Verbalase** [comment]
no self inserts -durk

**Comment by BeeRobynn** [comment]
> chat didnt we literally remove plyush's plushie over self-inserts

I wanted to have the plushie be admin spawn only so it could be spawned as a funny joke

**Pull request merged**